quilt3_local/pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "quilt3_local"
+version = "1.0.0-beta.11"
+description = ""
+authors = ["Alexei Mochalov <alexei@quiltdata.io>"]
```

Comment:
I think we should use the same as here:
https://github.com/quiltdata/quilt/blob/12fab1fb5e79b9cdfaddca1d31a76bbe47778df7/api/python/setup.py#L55-L56